### PR TITLE
chore: Spelling comments

### PR DESCRIPTION
**Pull request by jsoref**
## What do these changes do?

Fixes spelling errors in comments per https://github.com/alibaba/GraphScope/pull/3547#issuecomment-1950933992

## Related issue number

### CODE REVIEW

File: .github/free-disk-space.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@
 # Free: 17 GB
 # This script frees up 28 GB of disk space by deleting unneeded packages and 
 # large directories.
-# The Flink end to end tests download and generate more than 17 GB of files,
+# The following end to end tests download and generate more than 17 GB of files,
```
Comment (sighingnow) [inline .github/free-disk-space.sh:25]: It should be `Flink` as the script is referred from the flink's repo.

**Review by sighingnow — APPROVED** [review]

File: .github/free-disk-space.sh (hunk 1 above)
Comment (jsoref) [reply .github/free-disk-space.sh:25]: Do you want me to rebase and drop the commit or revert? (or something else?)

Comment (sighingnow) [reply .github/free-disk-space.sh:25]: You could revert the commit that touched this line.

**Comment by sighingnow** [comment]
Merged. Thank you, @jsoref!

**Pull request merged**